backend/manager/modules/vdsbroker/src/main/java/org/ovirt/engine/core/vdsbroker/VmManager.java

```diff
@@ -169,6 +173,9 @@ public Lock getVmDevicesLock() {
 
     public void update(VmDynamic dynamic) {
         vmDynamicDao.update(dynamic);
+        if (cpuPinningPolicy == CpuPinningPolicy.RESIZE_AND_PIN_NUMA) {
+            numOfCores = dynamic.getCurrentSockets() * dynamic.getCurrentCoresPerSocket();
+        }
```

Comment:
so would numOfCores be 0 when starting the engine while the a vm with resize-and-pin already runs?